src/test/java/com/aws/greengrass/clientdevices/auth/configuration/CAConfigurationTest.java

```diff
@@ -44,56 +46,55 @@ void afterEach() throws IOException {
     }
 
     @Test
-    public void GIVEN_cdaDefaultConfiguration_WHEN_getCATypeList_THEN_returnsEmptyList() {
+    public void GIVEN_cdaDefaultConfiguration_WHEN_getCATypeList_THEN_returnsEmptyList() throws URISyntaxException {
```

Comment:
All these changes are just to declare that it might throw an exception if you provide and invalid URI